website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -346,94 +346,101 @@ In the configuration format for the model SQL file:
 
 ## Dynamic tables
 
-The Snowflake adapter supports [dynamic tables](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table).
+The Snowflake adapter supports [dynamic tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about).
 This materialization is specific to Snowflake, which means that any model configuration that
 would normally come along for the ride from `dbt-core` (e.g. as with a `view`) may not be available
 for dynamic tables. This gap will decrease in future patches and versions.
 While this materialization is specific to Snowflake, it very much follows the implementation
 of [materialized views](/docs/build/materializations#Materialized-View).
 In particular, dynamic tables have access to the `on_configuration_change` setting.
-There are also some limitations that we hope to address in the next version.
+Dynamic tables are supported with the following configuration parameters:
 
-### Parameters
+| Parameter                 | Type     | Required | Default   | Change Monitoring Support | Reference                                    |
+|---------------------------|----------|----------|-----------|---------------------------|----------------------------------------------|
+| `on_configuration_change` | STRING   | NO       | `'apply'` | N/A                       |                                              |
+| `target_lag`              | STRING   | YES      |           | ALTER                     | [Target lag](#target-lag)                    |
+| `snowflake_warehouse`     | STRING   | YES      |           | ALTER                     | [Warehouse](#configuring-virtual-warehouses) |
```

Comment:
The table can be one column thinner if doing a hyperlink like [`snowflake_warehouse`](#configuring-virtual-warehouses) in the first column instead.